package.json

```diff
@@ -7,22 +7,12 @@
     "test": "test"
   },
   "dependencies": {
-    "babel-polyfill": "^6.26.0",
-    "classnames": "^2.2.5",
-    "lodash": "^4.17.11",
-    "patternfly": "^3.58.0",
-    "patternfly-react": "^2.25.4",
-    "prop-types": "^15.6.2",
-    "react": "^16.6.3",
-    "react-bootstrap": "^0.32.1",
-    "react-dom": "^16.6.3",
-    "react-json-tree": "^0.11.0",
-    "reselect": "^3.0.1",
-    "seamless-immutable": "^7.1.3"
+    "@theforeman/vendor": "^0.1.1",
```

Comment:
I suppose this is only available with Foreman 1.23?

Reply:
Right